### PR TITLE
metro stop number fix

### PR DESCRIPTION
**Pull request by Divinare**
Added stop number to metro-ats.proto

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>fi.hsl</groupId>
     <artifactId>transitdata-common</artifactId>
-    <version>1.2.2</version>
+    <version>1.2.3</version>
```
Comment (kenkus-futurice) [inline pom.xml]: Let's not increase the version in feature branch.

File: src/main/java/fi/hsl/common/mqtt/proto/Mqtt.java [hunk 2]
```diff
@@ -147,7 +147,7 @@ private RawMessage(
      * <code>required int32 SchemaVersion = 1 [default = 1];</code>
```
Comment (kenkus-futurice) [inline src/main/java/fi/hsl/common/mqtt/proto/Mqtt.java]: Why this file has changed?

File: src/main/java/fi/hsl/common/transitdata/proto/PubtransTableProtos.java [hunk 3]
```diff
@@ -20,15 +20,15 @@ public static void registerAllExtensions(
```
Comment (kenkus-futurice) [inline src/main/java/fi/hsl/common/transitdata/proto/PubtransTableProtos.java]: Why this file has changed?

File: src/main/java/fi/hsl/common/mqtt/proto/Mqtt.java (hunk 2 above)
Comment (Divinare) [reply src/main/java/fi/hsl/common/mqtt/proto/Mqtt.java]: No idea, maybe generate protos uses some old version of something. Reverting that file.

File: src/main/java/fi/hsl/common/transitdata/proto/PubtransTableProtos.java (hunk 3 above)
Comment (Divinare) [reply src/main/java/fi/hsl/common/transitdata/proto/PubtransTableProtos.java]: No idea, maybe generate protos uses some old version of something. Reverting that file.

**Pull request merged**